File: TFSWay.DomainModel/Entities/Query.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace TFSWay.DomainModel.Entities
{
   public class Query
    {
        [Key]
        public int QueryId { get; set; }
        public int ProjectID { get; set; }
        public string Subject { get; set; }
        public string AssignTo { get; set; }
        public string AssignToMailID { get; set; }
        public string Querytype { get; set; }
        public string Severity { get; set; }
        public string Status { get; set; }
        public string RefDocument { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string CreatedBy { get; set; }
    }
}

File: TFSWayUI/src/app/query/query.ts
export interface IQuery {

    queryId: number;
    ProjectID: number;
    QueryText: string;
    ReplyText: string;
    AssignTo: string;
    assignToMailID: string;
    Documnets: string;
    DocumnetPath: string;
    RefDocument: string;
    CreatedDate: Date;
    CreatedBy: string;


    queries: Array<IQuery>;

}

